test/unit/disable-upgrade.html

```diff
@@ -238,30 +242,39 @@ <h2 id="element">[[prop]]</h2>
         assert.ok(el.$.enabledEl.enabled);
         assert.ok(el.$.enabledEl.$.element);
         assert.equal(el.$.enabledEl.$.element.textContent, 'enabled!');
+        el.$.enabledEl.fooHandler = sinon.spy();
+        el.$.enabledEl.fire('foo');
+        assert.equal(el.$.enabledEl.fooHandler.callCount, 1);
         assert.notOk(el.$.disabledEl.hasCreated);
         assert.notOk(el.$.disabledEl.enabled);
         assert.notOk(el.$.disabledEl.$);
+        el.$.disabledEl.fooHandler = sinon.spy();
+        el.$.disabledEl.fire('foo');
+        assert.equal(el.$.disabledEl.fooHandler.callCount, 0);
         assert.notOk(el.$.disabledBoundEl.hasCreated);
         assert.notOk(el.$.disabledBoundEl.enabled);
         assert.notOk(el.$.disabledBoundEl.$);
+        el.$.disabledBoundEl.fooHandler = sinon.spy();
+        el.$.disabledBoundEl.fire('foo');
+        assert.equal(el.$.disabledBoundEl.fooHandler.callCount, 0);
       });
 
       test('elements upgrade when `disable-upgrade` removed', function() {
-        assert.notOk(el.$.disabledEl.hasCreated);
```

Comment:
Can we not remove these assertions or are they regressions? Are they a breaking change for our users?

Reply:
These were actually duplicate assertions from the previous test (https://github.com/Polymer/polymer/pull/5418/files#diff-ed6f0a354a50488549e9b8ea17788d86R240) that were incorrectly being tested again.